src/content/docs/query-your-data/explore-query-data/get-started/introduction-querying-new-relic-data.mdx

```diff
@@ -21,46 +21,101 @@ redirects:
 
-* [Query builder in NRQL mode](/docs/chart-builder/use-chart-builder/choose-data/use-advanced-nrql-mode-specify-data): query using [New Relic query language](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language) (NRQL), the same language we use to build most of our UI experiences.
-* [Query builder in PromQL-style mode](/docs/query-your-data/explore-query-data/chart-builder/use-advanced-promql-mode-specify-data): write basic queries using a PromQL-style query.
-
-### Query using simple UI interfaces [#nrqlless-query]
-
-[New Relic One](/docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one) offers several experiences that don't require knowledge of NRQL:
-
-* [Data explorer](/docs/query-your-data/explore-query-data/data-explorer/introduction-data-explorer): an intuitive data navigator to create visualizations for [events](/docs/using-new-relic/data/understand-data/new-relic-data-types#event-data) and [metrics](/docs/using-new-relic/data/understand-data/new-relic-data-types#metrics).
-* [Query builder (basic mode)](/docs/chart-builder/use-chart-builder/choose-data/use-basic-mode-specify-data): a simple, query-less, chart-building experience.
-* [Distributed tracing query](/docs/understand-dependencies/distributed-tracing/ui-data/additional-distributed-tracing-features-new-relic-one): a specialized UI for querying traces.
-* [Logs query](/docs/logs/new-relic-logs/ui-data/query-syntax-logs): a specialized UI for querying [New Relic Logs](/docs/logs/new-relic-logs/get-started/introduction-new-relic-logs) data.
+* [Query builder in NRQL mode](/docs/chart-builder/use-chart-builder/choose-data/use-advanced-nrql-mode-specify-data): Query using [New Relic query language](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language) (NRQL), the same language we use to build most of our UI experiences, and the most advanced way of querying data in New Relic.
+* [Query builder in PromQL-style mode](/docs/query-your-data/explore-query-data/chart-builder/use-advanced-promql-mode-specify-data): Write basic queries using a PromQL-style query.
 
 ## Query data via API [#query-apis]
 
-You can use APIs to retrieve and query your data in New Relic. For example, you can run [NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language) (our query language) queries with [NerdGraph](/docs/apis/graphql-api/get-started/introduction-new-relic-nerdgraph) (our GraphQL API).
+For use cases where getting into the New Relic platform is not an option, you can use APIs to retrieve and query your data in New Relic. For example, you can run [NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language) (our query language) queries with [NerdGraph](/docs/apis/graphql-api/get-started/introduction-new-relic-nerdgraph) (our GraphQL API).
```

Comment:
I'd rephrase this to `When getting into the New Relic platform is not an option,`. This way we're more direct.

Reply:
Good call, thanks!